<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RailTrackModellingToolbox</class>
 <widget class="QMainWindow" name="RailTrackModellingToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Railtrack Modelling Toolbox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MultiSleeperBtn">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>488</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi sleeper model GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ThreeSleeperBtn">
    <property name="geometry">
     <rect>
      <x>776</x>
      <y>239</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Three sleeper model GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImpulseModelBtn">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>568</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Impulse Model GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SemiAnalyticalTrackBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Semi-analytical Track Model GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PadStiffnessBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pad Stiffness GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SalomeBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Salome-Meca FE Pre-Post</string>
    </property>
   </widget>
   <widget class="QLabel" name="backGrndImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1071</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>RailTrackModellingToolbox.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>backGrndImage</zorder>
   <zorder>MultiSleeperBtn</zorder>
   <zorder>ThreeSleeperBtn</zorder>
   <zorder>ImpulseModelBtn</zorder>
   <zorder>SemiAnalyticalTrackBtn</zorder>
   <zorder>PadStiffnessBtn</zorder>
   <zorder>SalomeBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
